index.js

```diff
@@ -1,13 +1,99 @@
+$("#select").val(localStorage.getItem("optionVal")); //local storage
 
-async function init () {
-    try {
-        const selectVal = $("#select").val()
-        const response = await getUsers({ url: `https://randomuser.me/api/?results=${selectVal}` })
-        const { results } = response
-    } catch (err) {
-        console.log(err)
-        alert(`message: ${err.statusText} , status: ${err.status}`)
-    }
+const mapping = {
+  firstName: { path: "name.first", isVisible: true },
+  lastName: { path: "name.last", isVisible: true },
+  city: { path: "location.city", isVisible: true },
+  address: { path: "location.street.name", isVisible: true },
+  src: { path: "picture.large", isVisible: true },
+  gender: { path: "gender", isVisible: true },
+  email: { path: "email", isVisible: true },
+};
+
+async function init() {
+  try {
+    const selectVal = localStorage.getItem("optionVal");
+    const response = await getUsers({
+      url: `https://randomuser.me/api/?results=${selectVal}`,
```

Comment:
should be function that recieved the value and not extract it straight from the UI